src/qt/walletcontroller.cpp

```diff
@@ -231,7 +232,11 @@ void CreateWalletActivity::askPassphrase()
 
 void CreateWalletActivity::createWallet()
 {
-    showProgressDialog(tr("Creating Wallet <b>%1</b>…").arg(m_create_wallet_dialog->walletName().toHtmlEscaped()));
+    showProgressDialog(
+        //: Title of progress window when Create Wallet action is selected.
+        tr("Create Wallet"),
+        //: Describing what is being done in background.
```

Comment:
```suggestion
        //: Descriptive text of the create wallet progress window which indicates to the user what wallet is currently being created.
```

Reply:
This translator comment description looks a lot better! Addressed in ccecfa8